README.md

```diff
@@ -984,7 +985,60 @@ Each topic has two identifiers:
 
 ## 4.3 File Services
 
-### 4.3.1 GET Files (Header) Service
+### 4.3.1 GET Project Files Service
+
+**Resource URL**
+
+    GET /bcf/{version}/projects/{project_id}/files
```

Comment:
Maybe the endpoint could be named to getFileInformation.

Reply:
@pasi-paasiala @GeorgDangl fixed in the latest commit, I think this pull request is ready to merge.